pkg/virt-api/webhooks/validating-webhook/admitters/vmclone-admitter.go

```diff
@@ -43,6 +43,11 @@ import (
 	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
 )
 
+const (
+	virtualMachineKind         = "VirtualMachine"
```

Comment:
These are also available in `staging/src/kubevirt.io/api/core/v1/register.go`

Reply:
Unfortunately:
These ~are~ should also available in `staging/src/kubevirt.io/api/core/v1/register.go`

@xpivarc Do you think we are missing these constants?
In case we should add them for every resources: "VirtualMachine", "VirtualMachineInstance", "VirtualMachineInstanceReplicaSet", ... and more other... :)

Reply:
AFAIK, K8s doesn't have a similar structure in https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/register.go.

For example, they don't have the `{KindName}GroupVersionKind` exported variables.
@xpivarc what is the reason we have them in `staging/src/kubevirt.io/api/core/v1/register.go`?

Reply:
Convenience. For example, see `NewControllerRef`

Reply:
Thank you